src/formatting/value.rs

```diff
@@ -13,7 +13,7 @@ pub struct Value {
 #[derive(Debug, Clone)]
 pub enum ValueInner {
     Text(String),
-    Icon(String),
+    Icon(String, Option<f64>),
```

Comment:
We can use `Cow<'static, str>` here since most of the time icon names are static

Reply:
Just added that in :cow:!